todolist/src/components/AddTodo.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from "react";
+import {addTodoAction} from '../actions/index';
+import {connect} from 'react-redux'
+import Todo from '../components/todo'
+
+
+class AddTodo extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      input: ""
+    };
+  }
+  inputChangeHandler = e => {
+    this.setState({ input: e.target.value });
+  };
+
+  submitHandler = e => {
+    e.preventDefault();
+    (this.props.addTodoAction(this.state.input));
```

Comment:
There's a method you can call here to `reset` your input `state` after submission.